Neos.Flow/Classes/Persistence/Doctrine/Service.php

```diff
@@ -696,7 +699,7 @@ protected function buildCodeFromSql(Configuration $configuration, array $sql): s
             array_unshift(
                 $code,
                 sprintf(
-                    '$this->abortIf($this->connection->getDatabasePlatform()->getName() !== %s, %s);',
+                    '$this->abortIf($this->connection->getDatabasePlatform()->getName() != %s, %s);',
```

Comment:
Is that change intentional? Because the variable given could also be `null`?

Reply:
No, should be  `!==` as per https://github.com/doctrine/migrations/blob/1.8/lib/Doctrine/DBAL/Migrations/Tools/Console/Command/DiffCommand.php#L147

Reply:
```suggestion
                    '$this->abortIf($this->connection->getDatabasePlatform()->getName() !== %s, %s);',
```